<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameCreationDemosMove</class>
 <widget class="QFrame" name="FrameCreationDemosMove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>251</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="LayoutFrameEchiquier"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="LayoutVerticalButtons">
      <item>
       <widget class="QPushButton" name="Button_ValidationConfig">
        <property name="text">
         <string>Validate new config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_SelectPieceMode">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Mode register auto move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_RegisterPlayerMove">
        <property name="text">
         <string>Mode register player move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_ChoiceXmlFile">
        <property name="text">
         <string>Choice XML file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_AddExplanation">
        <property name="text">
         <string>Add explanation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_CreateXml">
        <property name="text">
         <string>Create XML File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
